wherehows-common/src/main/java/wherehows/common/schemas/DatasetPropertiesRecord.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package wherehows.common.schemas;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class DatasetPropertiesRecord extends AbstractRecord {
```

Comment:
The class name is very generic, yet the usage and content are limited to inventory refresh use case.
Maybe we can consider DatasetInventoryBasicPropertiesRecord or DatasetInventoryPropertiesRecord instead.